### PR TITLE
2022-sefsc-summer Champions post

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

File: content/blog/2022-sefsc-summer.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+
+### Planning improvements for reports
+
+Reports and analyses for Integrated Ecosystem Assessment (IEA) are often recurring or require edits to data inputs throughout the process. These workflows are spread throughout individuals' Google folders and collaborators have been communicating via multiple email threads.  The Openscapes Cohort helped this team to envision a better future with open science tools including data extraction code stored on GitHub that feeds directly into Quarto reports (see figure below). They also envision tracking current data versions with auto-generated date tags, an improvement over many versions of the same product in individual workflows that are difficult to track. The IEA team documented this process and improvements for the SEFSC 2022 Summer Cohort; many teams at SEFSC are working through similar issues and striving towards the highest quality data and most efficient generation of reports. All teams found common ground in this effort, highlighting positive impacts of open science practices that facilitate inspiration, expand collaboration, and synergize our diverse expertise across the Center.
+
+IMAGE HERE
```
Comment (jules32) [inline content/blog/2022-sefsc-summer.md]: This should be the sefsc-fates-pathway.png

Comment (jules32) [reply content/blog/2022-sefsc-summer.md]: (but it should maybe be renamed sefsc-iea-pathway.png

File: content/blog/2022-sefsc-summer.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+<br><center><a><img src="/img/blog/sefsc-fates-commsplan.png" width="900px" alt="alt txt"></a><figcaption>FATES-BLH Branch team communication plan which highlights current successes and areas for improvement.</figcaption></center><br>
+
+<br>
+
+<br> <center><a><img src="/img/blog/sefsc-fates-pathway.png" width="900px" alt="alt txt"></a><figcaption>FATES-BLH Openscapes team in their new pathway!  SEFSC Staff are spread throughout multiple cities in the southeast region, but Openscapes has facilitated streamlined communication for all staff.</figcaption></center><br>
```
Comment (jules32) [inline content/blog/2022-sefsc-summer.md:48]: This png belongs below (next comment) - this one is the pathway illustration for FATES.

**Review by jules32 — COMMENTED** [review]
Hi Stef! Things look good to me, just two things with the images

**Comment by stefaniebutland** [comment]
ack. Thanks @jules32 !!

**Comment by stefaniebutland** [comment]
draft tweet; lame so far

Read about the @openscapes Champions from @NOAAFisheries Southeast "Aligning mission around collaborative practices, with researchers, supervisors, and IT" by
@Molly304 @adyadyan @erica_rule
LINK

**Comment by AdyanRios-NOAA** [comment]
Molly is @molly304

**Comment by AdyanRios-NOAA** [comment]
I just asked if she want you to use it and she said yes 👍🏼

**Comment by AdyanRios-NOAA** [comment]
I like corny posts!! Something like "a growing wave of open science champions among fisheries scientists in the southeast" @stefaniebutland

**Comment by stefaniebutland** [comment]
thanks Ady! I'm so glad GitHub tagged you when I used your twitter handle :-)

**Comment by stefaniebutland** [comment]
@jules32 any edits to draft tweet?

Read about the growing wave of open science champions among @NOAAFisheries scientists in the southeast in this post by @Molly304 @adyadyan @erica_rule
https://www.openscapes.org/blog/2022/09/30/2022-sefsc-summer/

**Comment by jules32** [comment]
love it! no comments!

**Pull request merged**